avatars/simulation/wind.js

```diff
@@ -24,6 +24,14 @@ const update = (timestamp, headPosition, springBoneManager) => {
       return -1;
     }
 
+    const _handleNormal = () =>{
+      for (const springBones of springBoneManager.springBoneGroupList) {
+        for (const o of springBones) {
+          o.gravityDir = new THREE.Vector3()
```

Comment:
the 'new' allocates memory, can we create a default and use it repeatedly

Reply:
Ah, I didn't think of that. I'll update soon. :)